src/components/Dashboard/Chart/Chart.container.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from "react-redux";
+import { getContract } from "../../../modules/contract/selectors";
+import Chart from "./Chart";
+export const mapState = (state) => {
+  return {
+    contract: getContract(state),
+  };
+};
+
+export const mapDispatch = (dispatch) => ({});
+
+export default connect(mapState, mapDispatch)(Chart);
```

Comment:
Also here, if mapDispatch is not needed it can be removed.

Reply:
Done!